src/components/NewProjectForm/index.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react'
+import errorFinder from '@helpers/errorFinder'
+import Errors from '@components/Errors'
+import Wrapper from '@components/Wrapper'
+import Button from '@components/Button'
+import InputWithLabel from '@components/InputWithLabel'
+import SelectDropdown from '@components/SelectDropdown'
+import { type ValidationError } from '@types/ValidationError'
+import { Label } from './styles'
+
+type Props = {
+  onSubmit: () => any,
+  locales: Array<{ value: string, text: string }>,
+  errors: Array<ValidationError>
+}
+
+const NewProjectForm = ({ locales, onSubmit, errors }: Props) => {
+  const findErrors = errorFinder(errors)
+  const nameErrors = findErrors('name')
+
+  return (
+    <form onSubmit={onSubmit}>
+      <Wrapper>
+        <InputWithLabel
+          label="Project name"
+          name="name"
+          placeholder="MyAppName"
+        />
+        <Errors name="name" errors={nameErrors} />
+      </Wrapper>
+      <Wrapper mTop="regular">
+        <Label htmlFor="locale">Default locale</Label>
+        <SelectDropdown
+          onApply={options => {
+            console.log('options', options)
+          }}
+          isMultiple={false}
+          placeholder="Please select a locale"
+          options={locales}
+        />
+      </Wrapper>
+      <Wrapper mTop="large">
+        <Button type="submit">Add project</Button>
```

Comment:
We could make sure the form is disabled until it has finished loading by using the `<fieldset>` trick done in login/register.
It’s probably missing other client-side validations too, can you compare?